src/components/Input/Input.styles.js

```diff
@@ -0,0 +1,153 @@
+/* stylelint-disable no-descending-specificity */
+import styled from "styled-components";
+import { constants, themes } from "../../theme";
+
+const errorColor = ({ theme: { themeName } }) => themes[themeName].error.base;
+const white = ({ theme: { themeName } }) => themes[themeName].white.base;
+const primaryColor = ({ theme: { themeName } }) =>
+  themes[themeName].primary.base;
+const gray01 = ({ theme: { themeName } }) => themes[themeName].gray01;
+const gray02 = ({ theme: { themeName } }) => themes[themeName].gray02;
+
+export const FieldInputWrapper = styled.div`
+  display: flex;
+  position: relative;
+  &.text--input-top {
+    flex-direction: column;
+    justify-content: left;
+  }
+  &.text--input-left {
+    flex-direction: row;
+  }
+  &.text--input-disabled {
+    pointer-events: none;
+    opacity: 0.4;
+  }
+`;
+
+export const FieldInputText = styled.label`
+  font-size: 12px;
+  height: 13px;
+  line-height: 1.25;
+  text-align: left;
+  color: ${gray01};
+  position: relative;
+  .text--input-small & {
+    font-size: 10px;
+    line-height: 1.3;
+  }
+  .text--input-large & {
+    font-size: 14px;
+    line-height: 1.29;
+  }
+  .text--input-disabled & {
+    opacity: 0.4;
+  }
+  .text--input-top & {
+    margin-bottom: 6px;
+  }
+  .text--input-left & {
+    margin-right: 11px;
+    margin-top: 10px;
+  }
+  .text--input-left.text--input-small & {
+    margin-top: 5px;
+  }
+  .text--input-left.text--input-large & {
+    margin-top: 14px;
```

Comment:
Can we use `font-size` and `spacing` constants here please?

Reply:
Agree concerning `font-size` but I don't think we need to use `spacing` as we don't want this padding to change once `spacing` values are changed.